packages/editor/src/components/post-publish-panel/maybe-upload-media.js

```diff
@@ -114,6 +116,7 @@ export default function PostFormatPanel() {
 									resolve();
 								},
 								onError() {
+									setHadUploadError( true );
```

Comment:
This only catches errors from `media upload`; we also need to catch errors for `window.fetch`.

Reply:
This promise chain is getting fun 😄

I added a catch-all to the end of each individual upload. That should catch any errors / rejections, whether they come from the `window.fetch` or one of the later steps.

How does that look?